### PR TITLE
[FEATURE] Integrate JS for 3D viewer

### PR DESCRIPTION
**Pull request by beatrycze-volk**
Depends on #686

### CODE REVIEW

**Comment by beatrycze-volk** [comment]
- toastify.js is a included library, I would advise just ignore errors in there
- main.js should now be without Codacy warnings

File: Resources/Public/Javascript/3DViewer/importmap.js
Comment (sebastian-meyer) [inline Resources/Public/Javascript/3DViewer/importmap.js]: Please add linebreak at end of file (according to coding guidelines).

File: Resources/Public/Javascript/3DViewer/main.js
Comment (sebastian-meyer) [inline Resources/Public/Javascript/3DViewer/main.js]: We should use relative paths here if possible.

File: Resources/Public/Javascript/3DViewer/main.js
Comment (sebastian-meyer) [inline Resources/Public/Javascript/3DViewer/main.js:15]: Please add comment with license information, authors and link to the viewer component's GitHub: https://github.com/slub/3d-viewer

File: Classes/Controller/AbstractController.php
Comment (sebastian-meyer) [inline Classes/Controller/AbstractController.php]:
```suggestion
                    // Make sure configuration PID is set when applicable
```
(see https://github.com/kitodo/kitodo-presentation/pull/803#discussion_r937722879)

**Review by sebastian-meyer — CHANGES_REQUESTED** [review]

**Comment by sebastian-meyer** [comment]
> * toastify.js is a included library, I would advise just ignore errors in there

Do we even need toastify? It seems to be used only to make debugging a little easier. If I am correct, we should discuss if we can remove it before merging into Kitodo.Presentation. Otherwise please move toastify to it's own directory (i. e. `Resources/Public/Javascript/Toastify/`). Then I'll exclude it from Codacy scans.

File: Resources/Public/Javascript/3DViewer/main.js
Comment (beatrycze-volk) [reply Resources/Public/Javascript/3DViewer/main.js:15]: Where exactly I will find info about license? Or is it the same license as Kitodo?

**Comment by beatrycze-volk** [comment]
> > * toastify.js is a included library, I would advise just ignore errors in there
> 
> Do we even need toastify? It seems to be used only to make debugging a little easier. If I am correct, we should discuss if we can remove it before merging into Kitodo.Presentation. Otherwise please move toastify to it's own directory (i. e. `Resources/Public/Javascript/Toastify/`). Then I'll exclude it from Codacy scans.

I have moved it to the own directory.

There is one error left in main.js but it is false positive:
'onProgress' was used before it was defined. - actually it is defined as the const in  in line 578, error appears in line 610.

File: Resources/Public/Javascript/3DViewer/main.js
Comment (sebastian-meyer) [reply Resources/Public/Javascript/3DViewer/main.js:15]: Since it is code from @thedworak, he should decide which license to use. In order to be able to integrate in Kitodo, it has to be a GPL-compatible license or the GPL itself. Could you please ask him for license information?

**Review by sebastian-meyer — DISMISSED** [review]
Apart from the missing license information in the main 3D viewer javascript file this is ready for merging.

**Review by sebastian-meyer — APPROVED** [review]

**Pull request merged**
